src/Symfony/Component/Serializer/Tests/Context/Normalizer/ProblemNormalizerContextBuilderTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Context\Normalizer;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Serializer\Context\Normalizer\ProblemNormalizerContextBuilder;
+use Symfony\Component\Serializer\Normalizer\ProblemNormalizer;
+
+/**
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ */
+class ProblemNormalizerContextBuilderTest extends TestCase
+{
+    private ProblemNormalizerContextBuilder $contextBuilder;
+
+    protected function setUp(): void
+    {
+        $this->contextBuilder = new ProblemNormalizerContextBuilder();
+    }
+
+    /**
+     * @dataProvider withersDataProvider
+     *
+     * @param array<string, mixed> $values
+     */
+    public function testWithers(array $values)
+    {
+        $context = $this->contextBuilder
+            ->withTitle($values[ProblemNormalizer::TITLE])
```

Comment:
As the constant is public, hardcoding value is better IMO, it will make this test break if the constant value change, same for other constant

Reply:
Yes I see your point, but as a constant is used either in normalizers/encoders and in context builders, changing the constant value won't break anything